src/tests/tribol_common_plane_gpu.cpp

```diff
@@ -98,7 +98,7 @@ double runExample( int num_elems_1d )
       break;
 #endif
 #ifdef TRIBOL_USE_HIP
-    case MemorySpace::Hip:
+    case MemorySpace::Device:
```

Comment:
Is this a refactor?

Reply:
No, it's a bug.  Once the tioga CI PR is merged, this sort of stuff will go away.